plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GenerateEnclaveMetadata.kt

```diff
@@ -13,9 +15,20 @@ open class GenerateEnclaveMetadata @Inject constructor(
     private val buildType: BuildType,
     private val linuxExec: LinuxExec
 ) : ConclaveTask() {
+    companion object {
+        const val ENCLAVE_MRSIGNER_FILE = "mrsigner.txt"
+        const val ENCLAVE_MRENCLAVE_FILE = "mrenclave.txt"
+    }
+
     @get:InputFile
     val inputSignedEnclave: RegularFileProperty = objects.fileProperty()
 
+    @OutputFile
+    val mrsignerOutputFile: RegularFileProperty = objects.fileProperty()
+
+    @OutputFile
```

Comment:
These need to have the `get:` prefix, otherwise don't work. See the Javadocs for more info.

Reply:
Also, any reason why two files and not one. We can add further metdata to the single file in future

Reply:
So the files can be quickly added to command lines like so `$(cat <filename>)`, we don't want any parsing to be necessary.